libpod/sqlite_state.go

```diff
@@ -64,15 +64,15 @@ func NewSqliteState(runtime *Runtime) (_ State, defErr error) {
 		return nil, fmt.Errorf("setting full fsync mode in db: %w", err)
 	}
 
-	if err := state.migrateSchemaIfNecessary(); err != nil {
-		return nil, err
-	}
-
 	// Set up tables
 	if err := sqliteInitTables(state.conn); err != nil {
 		return nil, fmt.Errorf("creating tables: %w", err)
 	}
 
+	if err := state.migrateSchemaIfNecessary(); err != nil {
```

Comment:
@mheon there's probably a better way but this change unblocked me.

Reply:
Hm. It needs to happen before table init to be safe, so we'll probably have to figure out how to ignore the table-does-not-exist error.

Reply:
Can do that later though